src/angular/toast/toast-service.ts

```diff
@@ -0,0 +1,19 @@
+import { inject, Injectable, Injector } from '@angular/core';
+import {
+  _SbbOverlayBaseService,
+  SBB_OVERLAY_DATA,
+  SbbOverlayRef,
+} from '@sbb-esta/lyne-angular/core/overlay';
+
+import { type SbbToast } from './toast';
+import { SbbToastContainer } from './toast-container';
+
+@Injectable({ providedIn: 'root' })
+export class SbbToastService extends _SbbOverlayBaseService<SbbToastContainer, SbbToast> {
+  constructor() {
+    const injector = inject(Injector);
+    const parentDialog = inject(SbbToastService, { optional: true, skipSelf: true })!;
```

Comment:
should we consider specifc (not dialog related) cont names?
```suggestion
    const parentToastService = inject(SbbToastService, { optional: true, skipSelf: true })!;
```